lib/features/interactive.js

```diff
@@ -0,0 +1,93 @@
+const MAX_RANDOM_PORT_ATTEMPTS = 20;
+
+var debug = require('debug')('docker-worker:features:interactive');
+var DockerExecServer = require('docker-exec-websocket-server').DockerExecServer;
+var http = require('http');
+var https = require('https');
+var slugid = require('slugid');
+var Promise = require('promise');
+var url = require('url');
+var fs = require('fs');
+
+export default class WebsocketServer {
+  constructor () {
+    this.path = '/' + slugid.v4();
+  }
+
+  async started (task) {
+    debug('creating ws server');
+    var httpsOpt = true; //set this somewhere else later
```

Comment:
I might be wrong here but I think all config options are added to ´task.runtime`. So we can just add a config option in `config/defaults.js` called `interactive: {ssl: true}` and then get it here using `task.runtime.interactive.ssl`.

This is likely a good option for many config options that we configure on a per worker level.